.github/workflows/TASKLIST_CARBONISATION.yml

```diff
@@ -0,0 +1,67 @@
+name: Check Tasklist Carbonization
+
+on: [push]
+
+jobs:
+  Exec:
+    name: Run visual regression
+    runs-on: ubuntu-latest
+    container:
+      image: mcr.microsoft.com/playwright:v1.32.3-focal
+      options: --user 1001:1000
+    steps:
+      - name: Checkout form-js
+        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3
+        with:
+          path: form-js
+      - name: Checkout Tasklist
+        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3
+        with:
+          repository: camunda/tasklist
+          token: ${{ secrets.ADD_TO_HTO_PROJECT_PAT }}
+          path: tasklist
+          ref: "test-form-js-int"
+      - name: Setup form-js cache
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c
+        with:
+          node-version: "16"
+          cache: "npm"
+          cache-dependency-path: ./form-js/package-lock.json
+      - name: Setup Tasklist cache
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c
+        with:
+          node-version: "19"
+          cache: "yarn"
+          cache-dependency-path: ./tasklist/client/yarn.lock
+      - name: Install form-js deps
+        run: npm ci
+        working-directory: ./form-js
+      - name: Build form-js
+        run: npm run build
+        working-directory: ./form-js
+      - name: Link form-js-viewer
+        run: yarn link
+        working-directory: ./form-js/packages/form-js-viewer
+      - name: Link form-js-viewer
+        run: yarn link "@bpmn-io/form-js-viewer"
+        working-directory: ./tasklist/client
+      - name: Install Tasklist deps
+        run: yarn
+        working-directory: ./tasklist/client
+      - name: Build Tasklist
+        run: yarn build:visual-regression
+        working-directory: ./tasklist/client
+      - name: Start server
+        working-directory: ./tasklist/client
+        run: yarn start:visual-regression &
+      - name: Copy form-js schema
+        run: cp ./form-js/packages/form-js-viewer/test/schemaWithAllComponents.json ./tasklist/client/e2e-playwright/bigForm.json
```

Comment:
Yeah, I wasn't really sure about this one. I wanted to gather your opinion on this.

The idea is basically to remind us of having an updated complete form on Tasklist, but then it would create a "chicken and egg" problem :sweat_smile: 

We have to remove it I guess, but still we need a process to remember to update the schemas on the Tasklist repo when we add a new form component

Reply:
Since we mostly use these tests on CI, is there maybe a way to store definition and screenshot on a shared basis? Maybe inside `team-hto` or something. We could then copy both in the respective folders beforehand.

This wouldn't work locally and we have to update the screenshot manually somewhere.

Another thing could be to not have this action as a mandatory check, but rather a warning + sending a notification in Slack or something.

Reply:
The screenshot is stored on the Tasklist side, you can check it on [this PR](https://github.com/camunda/tasklist/pull/2869/files)
I think it's simpler to just store it there because then we wouldn't need to add any other automation to copy

We could add an automation to that ask the PR creator if it's adding a new component or not. If he answer yes then we would have to update the screenshot on the Tasklist side when we update `form-js`

Reply:
Agreed, let's add a PR template for it, then. 👍 

I think we're currently using the [shared bpmn-io template](https://github.com/bpmn-io/.github/blob/master/.github/PULL_REQUEST_TEMPLATE.md), we'd have to push another one.